src/emcpy/plots/plots.py

```diff
@@ -105,3 +105,45 @@ def __init__(self, x, y):
         self.markersize = None
         self.alpha = None
         self.label = None
+
+
+class VerticalLine:
+
+    def __init__(self, x):
+        """
+        Constructor for VerticalLine
+
+        Args:
+            x : (int/float) x-value where vertical line
+                is to be plotted
+        """
+
+        self.plottype = 'vertical_line'
+
+        self.x = x
+
+        self.color = 'black'
```

Comment:
I wonder, would be useful to have optional arguments in the constructor for color, linestyle, linewidth, and label, which would default to these settings?

Reply:
So, @kevindougherty-noaa is keeping this consistent with what he already has for the other classes.

Which for this application I can see the utility in doing:
```
lineobj = VerticalLine(x, color='red')
```
as opposed to
```
lineobj = VerticalLine(x)
lineobj.color = 'red'
```

Do we want all classes to have the attributes as optional arguments to the constructor also?

Reply:
@EdwardSafford-NOAA These are defaults, but because we are creating an object, they can be adjusted easily:
```
# Create plot object
plotobj = VerticalLine(x)
plotobj.color = 'red'
plotobj.linewidth = 5
plotobj.linestyle = '--'
```

Reply:
@CoryMartin-NOAA I argue against having optional arguments for the constructor because then you lose the declarative plotting method we have been aiming for. I think we should create the object with set defaults and then let the user change them declaratively if wanted.

Reply:
@kevindougherty-noaa well I think we `could` have both, but I don't know if it is necessary
```
lineobj = VerticalLine(x, color='red')
lineobj.linewidth = 5
```